.github/actions/argus-builder/build-prep/action.yml

```diff
@@ -126,4 +145,7 @@ runs:
         script: |
           const branchMatched = `${{ steps.branch_filter.outputs.match }}` === 'true';
           const filesMatched = `${{ steps.file_filter.outputs.run_on }}` === 'true';
-          core.setOutput('should_build', filesMatched && branchMatched);
+          const prHasStackLabel = `${{ steps.require_stack_label.outputs.has_trigger_label }}` === 'true';
+          const shouldBuild = filesMatched && branchMatched && prHasStackLabel;
+          core.setOutput('should_build', shouldBuild);
```

Comment:
it might be good to have the image still get built, the benefits of that would be even without making a stack a user would still be able to see whether their code introduced a bug that prevents the image from building. @jakeyheath what do you think?

Reply:
I think given that the main artifact for a build is often times an image, and given the low cost of building an image, we should just always build an image. what do we think about removing `should_build` from  build-docker's `if` condition?

```
if: needs.prep.outputs.should_build == 'true' && needs.prep.outputs.images != '[]'
```

alternatively I can create a new output like `shouldDeploy` and use that to separately control `Update manifest`

Reply:
yea that would be more clear @wontonst . Let's go forward with that